content/seminars/2020-06-19

```diff
@@ -0,0 +1,4 @@
+title: Use of Neural Networks to Improve Fuzzing: Charalambos Mitropoulos date: 2020-06-19 category: seminars
+
+Abstract: In this short seminar, after we present our recent work in fuzzing Android Native Libraries,
```

Comment:
Plase make this more concrete and check the syntax.  No need for "In this short seminar"